build/helper/documentation_helper.py

```diff
@@ -296,15 +296,20 @@ def _fix_references(doc, cfg, make_link=False):
     return doc
 
 
-def _format_type_for_rst_documentation(param, config):
-    p_type = param['python_type']
-    if param['enum'] is not None:
+def format_type_for_rst_documentation(param, config, numpy):
```

Comment:
Usually config goes at the end.
Applies to several places.

Reply:
Moved